frontend/fingertips-frontend/components/pages/search/results/index.module.css

```diff
@@ -0,0 +1,18 @@
+.result {
+  padding-inline-start: 0;
+}
+
+.result li {
+    border-bottom: 1px solid;
+    list-style: none;
+    padding: 2% 5% 0;
+}
+
+.result li:first-child {
+  border-top: 1px solid
+}
+
+.result li p {
+  font-size: 15px;
+  line-height: 10%;
+}
```

Comment:
Can we try and not use any css styles. The govuk-react component library for Paragraph contains modifiers that we can use to set the size. 

If there is any spacing we need to do. Try and used the styled-components way. An example is on 
`frontend/fingertips-frontend/components/forms/SearchForm/index.tsx`
```
const StyledInputField = styled(InputField)(
  spacing.withWhiteSpace({ marginBottom: 6 })
);
```

There is a component for the line known as section break https://govuk-react.github.io/govuk-react/?path=/docs/section-break--docs

Also an unordered list which we can use https://govuk-react.github.io/govuk-react/?path=/docs/unordered-list--docs so we don't need to style it. 

Hopefully using these components should make the css changes we need to make non-existent or minimal.

Happy to discuss

Reply:
updated